Detectors/Base/include/DetectorsBase/Utils.h

```diff
@@ -7,30 +7,32 @@
 
 #include "DetectorsBase/Constants.h"
 #include <math.h>
+#include <array>
+using std::array;
```

Comment:
This is against our coding guidelines:

https://htmlpreview.github.io/?https://github.com/AliceO2Group/CodingGuidelines/blob/master/coding_guidelines.html#Using_declarations_and_directives